openwisp_monitoring/monitoring/migrations/0021_resources_threshold_values.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.0.3 on 2020-08-17 10:18
+
+from django.db import migrations
+
+
+resources_metrics = {'memory': 95, 'disk': 80, 'cpu': 90}
```

Comment:
take the value from the configuration please, avoid duplication or we'll face issues if those values are changed.